packages/polaris-schema/src/utils/merge-types.ts

```diff
@@ -2,17 +2,24 @@ import { ITypeDefinitions } from 'graphql-tools';
 import { mergeTypes } from 'merge-graphql-schemas';
 import { repositoryEntityTypeDefs } from '../common/type-defs/repository-entity-type-defs';
 import { directivesTypeDefs } from '../directives/directives-type-defs';
-import { scalarsTypeDefs } from '../scalars/scalars-type-defs';
+import {
+    defaultPolarisScalarsTypeDefs,
+    polarisScalarsTypeDefs,
+} from '../scalars/scalars-type-defs';
 import { pageInfoTypeDef } from '../common/type-defs/page-info-type-def';
 import { onlinePagingInputTypeDefs } from '../common/type-defs/online-paging-type-defs';
 import { filtersTypeDefs } from '../common/type-defs/filters-type-defs';
+import { PolarisSchemaConfig } from '../config/polaris-schema-config';
 
-export const getMergedPolarisTypes = (types: ITypeDefinitions): string =>
-    mergeTypes(
+export const getMergedPolarisTypes = (
+    polarisSchemaConfig: PolarisSchemaConfig,
+    types: ITypeDefinitions,
+): string => {
+    const directivesAndScalarsTypeDefs = getDirectivesAndScalarsTypeDefs(polarisSchemaConfig);
+    return mergeTypes(
         [
+            ...directivesAndScalarsTypeDefs,
             repositoryEntityTypeDefs,
-            scalarsTypeDefs,
-            directivesTypeDefs,
             pageInfoTypeDef,
```

Comment:
check if this 3 is mandory, cause u didn't make a configuration for them
pageInfoTypeDef,
onlinePagingInputTypeDefs
filtersTypeDefs